projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts

```diff
@@ -308,4 +308,11 @@ export class AnnotationSetComponent implements OnInit, OnDestroy {
       width: width
     };
   }
+
+  annotationSetClass() {
```

Comment:
you could possibly create a test for this, as codecov is complaining

Reply:
:+1: